packages/react-components/src/components/chart/utils/getTrendCursor.ts

```diff
@@ -95,19 +93,27 @@ const ondragUpdateTrendCursorElements = ({
     ),
   ];
 };
-const ondragUpdateTrendCursor = ({ graphic, event, timeInMs, series, size, yMin, yMax }: ondragUpdateGraphicProps) => {
+export const ondragUpdateTrendCursor = ({
```

Comment:
Should the naming in this file not be `onDrag...` instead of `ondrag...`?

Reply:
fair call, will update